src/wp-includes/block-supports/layout.php

```diff
@@ -630,7 +630,16 @@ function wp_render_layout_support_flag( $block_content, $block ) {
 
 	$class_names        = array();
 	$layout_definitions = wp_get_layout_definitions();
-	$container_class    = wp_unique_id( 'wp-container-' );
+
+	/*
+	 * We use an incremental ID that is independent per prefix to make sure that
```

Comment:
Avoid using "We" words in Core, as its not well understood who "we" refers to.

```suggestion
	 * Uses an incremental ID that is independent per prefix to make sure that
```